docs/src/content/docs/de/guides/i18n.mdx

```diff
@@ -188,16 +188,16 @@ Du kannst Übersetzungen für zusätzliche Sprachen, die du unterstützt, über
 
 <Steps>
 
-1. Konfiguriere die `i18n` Datensammlung in `src/content/config.ts`, wenn sie nicht bereits konfiguriert ist:
+1. Konfiguriere die `i18n` Datensammlung in `src/content.config.ts`, wenn sie nicht bereits konfiguriert ist:
 
-   ```diff lang="js" ins=/, (i18nSchema)/
-   // src/content/config.ts
-   import { defineCollection } from 'astro:content';
+   ```diff lang="js" ins=/, (i18nLoader|i18nSchema)/
+   // src/content.config.ts
+   import { docsLoader, i18nLoader } from '@astrojs/starlight/loaders';
```

Comment:
Just opened #2707 because I don’t think we should have removed the `defineCollection` import here. Might as well update that right away in this PR too:

```suggestion
   import { defineCollection } from 'astro:content';
   import { docsLoader, i18nLoader } from '@astrojs/starlight/loaders';
```